### ISSUE TITLE
Color Scheme Doesn't Work Inside Repeater

### ISSUE THREAD

**Issue by premanshup** [elementor/elementor#3579]
## Description

When you add color control inside repeater and set scheme_color to COLOR_1 or COLOR_2 or COLOR_3 or COLOR_4, it doesn't set any default scheme color to that class or element. In fact, after adding color scheme to repeater we were unable to see the css for scheme color for that element in frontend. There was no css generated for color scheme but when you change the color from color picker in frontend, it reflects on the frontend and works properly.

Below is my code :-
```PHP
// Single Business Day Color.
$repeater->add_control(
	'single_business_day_color',
	[
		'label'     => __( 'Day Color', 'uael' ),
		'type'      => Controls_Manager::COLOR,
		'scheme'    => [
			'type'  => Scheme_Color::get_type(),
			'value' => Scheme_Color::COLOR_2,
		],
		'selectors' => [
			'{{WRAPPER}} {{CURRENT_ITEM}} .uael-business-day-highlight' => 'color: {{VALUE}}',
		],
		'condition' => [
			'highlight_this' => 'yes',
		],
		'separator' => 'before',
	]
);
```

Screenshot for default color :- 

![default-color](https://user-images.githubusercontent.com/36841355/36663105-86cbfffc-1b06-11e8-860e-82a57c148a6c.png)

![frontend](https://user-images.githubusercontent.com/36841355/36663128-955597f4-1b06-11e8-9e83-fe8f021bf21d.png)

## Steps to reproduce

1. Write code for scheme color - Scheme_Color::COLOR_2 inside REPEATER.
2. Go to frontend and open edit with elementor.
3. Add element in a section or column for which you have written code for color scheme inside repeater. 
4. Now set the color to clear. It should take the value of color scheme COLOR_2 but it will not.

## Environment

					
== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.29 (Win32) OpenSSL/1.1.0g PHP/7.2.1
	MySQL version: 5.5.5
	PHP Version: 7.2.1
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.4
	Site URL: http://localhost/timeline
	Home URL: http://localhost/timeline
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 40M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Active

== Theme ==
	Name: Storefront
	Version: 2.2.8
	Author: Automattic
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.186 Safari/537.36

**Comment by atashinbar** [comment]
Hi,
Your problem is solved? or not?

**Comment by sweethoneycode** [comment]
Is this using the development guide for the repeater control?
https://developers.elementor.com/elementor-controls/repeater-control/

Is it possible to post your full code into a gist?

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.0

Schemes are deprecated, replaced with Global Colors and Fonts

Feel free to update

Thanks!
